firstProject/diff-canvas.js

```diff
@@ -0,0 +1,295 @@
+
+    <canvas id="myCanvas" width="1300" height="700"></canvas>
+    <script>
+      var canvas = document.getElementById('myCanvas');
+      var context = canvas.getContext('2d');
+
+      context.beginPath();
+      context.arc(80,25,5,0,2*Math.PI);
+      context.stroke()
+      context.fillStyle = 'red'
+      context.fill()
+
+       context.beginPath();
+      context.moveTo(1,1)
+      context.lineTo(200,200)
+      context.quadraticCurveTo(230,220,310,200)
+      context.lineTo(400,200)
+      context.lineTo(600,200)
+      context.quadraticCurveTo(700,0,310,200)
+      context.fill()
+      context.stroke()
+      context.beginPath()
+      context.moveTo(700,200)
+      context.lineTo(600,200)
+      context.quadraticCurveTo(800,0,420,300)
+      context.stroke()
+      context.fill()
```

Comment:
Factor each one of these shapes into a function that draws that shape, given the starting x,y and the size, etc